content/en/docs/migration/ot_overview.md

```diff
@@ -0,0 +1,121 @@
+you can manage exporting data in multiple formats.
+
+Once you have the OpenTelemetry SDK installed, confirm that you can deploy your application and still
+recieve the same OpenTracing-based telemetry. In other words, corfirm that your dashboards, alerts, and other
+tracing-based analysis tools are still working.
+
+### Step 2: progressively replace instrumentation
+
+Once the OpenTelemetry SDK is installed, all new instrumentation can now be written using the OpenTelemetry API. With few
+exceptions, OpenTelemetry and OpenTracing instrumentation will work together seamlessly (see "limits on compatibility" below).
+
+What about existing instrumentation? There is no hard requirement to migrate existing application code to OpenTelemetry. 
+However, we do recommend migrating from any OpenTracing instrumentation libraries – libraries used to instrument web frameworks, HTTP clients, database clients, etc. – to their OpenTelemetry equivalents. This will improve support, as many OpenTracing libraries will be end-of-lifed and
+may no longer be updated.
+
+It is important to note that when switching to an OpenTelemetry instrumentation library, the data which is produced will change. OpenTelemetry has an improved model for how we instrument software (what we refer to as our "semantic conventions"). In many cases OpenTelemetry produces better, more comprehensive tracing data. However, "better" also means "different." This means that existing dashboards, alerts, etc. based on older OpenTracing instrumentation libraries may no longer work when those libraries are replaced.
+
+For existing instrumentation, it is recomended that 
+1. Replace one piece of OpenTracing instrumentation with its OpenTelemetry equivalent.
+2. Observe how this changes the telemetry which your application produces.
+3. Create new dashboards, alerts, etc which consume this new telemetry. Set up these dashboards before
+deploying the new OpenTelemetry library to production.
+4. Optionally, add proccessing rules to the Collector which convert the new telemetry back into the old telemetry.
+The Collector can then be configured to emit both versions of the same telemetry, creating a data overlap. This allows
+new dashboards to populate themselves while you continue to use the old dashboards.
+
+## Limits on compatibility
+
+When migrating to OpenTelemetry, there are several notable edge cases to be aware of.
+
+### Semantic Conventions are not the same
+As mentioned above in the migration guide, OpenTelemetry has an improved model for instrumenting software. This means that the "tags" which are set
+by OpenTracing instrumentation may be different than the "attributes" which are set by OpenTelemetry. In other words, when replacing existing instrumentation, the data OpenTelemetry produces may be different from the data OpenTracing produces. When switching instrumentaiton, be sure to also update any dashboards, alerts, etc. which relied on the old data.
```

Comment:
The end part of this paragraph seems redundant relative to what was shared earlier.

Reply:
Ok. I would like to hammer this point home, but I'll try to word it in a way that acknowledges that it has been said repeatedly.

Reply:
please check out the new wording